torch/_inductor/config.py

```diff
@@ -1193,6 +1193,11 @@ class rocm:
     # Install with `pip install git+https://github.com/rocm/composable_kernel@develop`.
     ck_dir = os.environ.get("TORCHINDUCTOR_CK_DIR")
 
+    # generate standalone executables for instances generated with the CK backend
+    generate_test_runner: bool = (
+        os.environ.get("INDUCTOR_CK_BACKEND_GENERATE_TEST_RUNNER_CODE", "1") == "1"
```

Comment:
I was wondering whether enabling this by default will significantly slowly down compile time since we'll have to do two compiles.

Reply:
Yes, mirroring the CUTLASS default here though. I can disable it